tests/io_perfomance/image-volume/run-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I highly recommend you check all your shell code with yetus/shellcheck @vk-en  . Is this on by default in Eden repo @mydatascience ?